inc/osvr/Client/RenderManagerConfig.h

```diff
@@ -0,0 +1,104 @@
+/** @file
+    @brief OSVR rendermanager configuration
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com>
+
+    */
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_RenderManagerConfig_h_GUID_C8DA5781_5B6C_454A_B4FF_1DB50CBE3479
+#define INCLUDED_RenderManagerConfig_h_GUID_C8DA5781_5B6C_454A_B4FF_1DB50CBE3479
+
+// Required for DLL linkage on Windows
+#include <osvr/Client/Export.h>
+#include <osvr/Util/UniquePtr.h>
+#include <osvr/Util/ClientOpaqueTypesC.h>
+
+// Standard includes
+#include <string>
+#include <iostream>
+#include <exception>
+#include <vector>
+
+namespace osvr {
+namespace client {
+
+    class RenderManagerConfig;
+    typedef unique_ptr<RenderManagerConfig> RenderManagerConfigPtr;
```

Comment:
who holds this `unique_ptr`? They aren't safe across DLL boundaries because the deletion function is generated in the usage scope. (Compare to `shared_ptr` where the deleter is captured and stored alongside the raw pointer at creation)